server/routes/cart.js

```diff
@@ -0,0 +1,90 @@
+const Cart = require("../models/Cart");
+const { route } = require("./auth");
+const { verifyToken, verifyTokenAndAuthorization, verifyTokenAndAdmin } = require("./verifyToken")
+
+const router = require("express").Router();
+
+router.post("/", verifyToken, async (req, res) => {
+    const newCart = new Cart(req.body)
+
+    try {
+        const savedCart = await newCart.save();
+        res.status(200).json(savedCart);
+    } catch(err) {
+        res.status(500).json(err)
+    }
+})
+
+//UPDATE
+router.put("/:id", verifyTokenAndAuthorization, async (req, res) => {
+  try {
+    const updatedCart = await Cart.findByIdAndUpdate(
+      req.params.id,
+      {
+        $set: req.body,
+      },
+      { new: true }
+    );
+    res.status(200).json(updatedCart);
+  } catch (err) {
+    res.status(500).json(err);
+  }
+});
+
+//DELETE
+router.delete("/:id", verifyTokenAndAuthorization, async (req, res) =>  {
+    try {
+        await Cart.findByIdAndDelete(req.params.id)
+        res.status(200).json("Cart has been deleted...")
+    } catch(err) {
+        res.status(500).json(err)
+    }
+})
+
+// GET USER CART
```

Comment:
Should we wait on cart functionality until we decide as a team that that's what we're going for? It wasn't in our mockups at all, so I don't know if we should necessarily go ahead on all this just yet (I think the cart idea is super cool, I just want to make sure we have like sketches and stuff first so we all know what this actually entails)

Reply:
Yup, I comment it out in the server.js, so we are not using the cart.js